docs/zenith/developer/go/457202-test-build-publish.md

```diff
@@ -0,0 +1,314 @@
+---
+slug: /zenith/developer/go/457202/test-build-publish
+displayed_sidebar: "zenith"
+toc_max_heading_level: 2
+authors: ["Erik Sipsma"]
+date: "2023-11-07"
+---
+
+# Test, Build and Publish an Application using a Module
+
+## Introduction
+
+This guide walks you through the process of creating a Dagger module from scratch to test, build and publish a Node.js application image. You will learn how to:
+
+- Initialize a new Dagger module and add dependencies to it
+- Write custom functions using the Go SDK to test, build and package an application
+- Use containers as function input arguments and return values
+- Debug module code using an interactive shell
+- Test module code by exposing services or containers returned by a function on the host
+
+## Requirements
+
+This guide assumes that:
+
+- You have a good understanding of the Go programming language and an IDE with Go language support.
+- You know the basics of programming Dagger modules. If not, refer to the [quickstart](./525021-quickstart.md).
```

Comment:
```suggestion
- You know the basics of running Dagger modules. If not, refer to the [quickstart](./525021-quickstart.md).
```

Reply:
In this change, you refer to "running modules" but the link points to the programming quickstart. Running modules is covered mainly in the user quickstart.  

The intent here is to have the user complete the programming quickstart first. Do you think we should also have as a requirement completion of the user quickstart? No objection to this in either case, but just trying to clarify your suggestion.

Reply:
Added links to both quickstarts